src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -352,6 +352,21 @@ private void OnPointerExited(Windows.UI.Core.PointerEventArgs args)
 
 		private void OnPointerPressed(Windows.UI.Core.PointerEventArgs args)
 		{
+			// If 2+ mouse buttons are pressed, we only respond to the first.
+			var buttonsPressed = 0;
+			var properties = args.CurrentPoint.Properties;
+			if (properties.IsLeftButtonPressed) { buttonsPressed++; }
+			if (properties.IsRightButtonPressed) { buttonsPressed++; }
+			if (properties.IsMiddleButtonPressed) { buttonsPressed++; }
+			if (properties.IsXButton1Pressed) { buttonsPressed++; }
+			if (properties.IsXButton2Pressed) { buttonsPressed++; }
+			if (properties.IsBarrelButtonPressed) { buttonsPressed++; }
+
+			if (args.CurrentPoint.PointerDeviceType == PointerDeviceType.Mouse && buttonsPressed > 1)
```

Comment:
What if we press 2 buttons out of the window, then move it over the window ? :P

Reply:
Come on now :D. I just tested and this works fine (as least on WPF). Generally, you're not supposed to be getting pointer events anyway if the pointer press started on another window.